src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -21,59 +20,34 @@ import { EmptyVulnerabilityData, GenericError } from '../../PresentationalCompon
 import DownloadReport from '../../../Helpers/DownloadReport';
 import CvePairStatusModal from '../Modals/CvePairStatusModal';
 import SystemCveTableToolbar from './SystemCveTableToolbar';
+import { useCreateUrlParams } from '../../../Helpers/MiscHelper';
 
 export const CVETableContext = createContext({});
 
-export const SystemCVEs = ({ intl, entity, location, history }) => {
+export const SystemCVEs = (props) => {
```

Comment:
why did you pass the `props` instead of the object? 
I think explicit is better than implicit.

Reply:
Because, I wanted to avoid useCreateUrlParams({ history, location }, CVES_ALLOWED_PARAMS) to achieve consistency with other pages.